dlp/api/Snippets/DeidentifyMasking.cs

```diff
@@ -18,7 +18,7 @@
 using Google.Cloud.Dlp.V2;
 using System;
 
-class DlpDeidentifyMasking
+public class DlpDeidentifyMasking
```

Comment:
So now I'm cofused, are these more or less the same as in the Samples project. Are the samples roughly duplicated in the Snippets vs. Samples project. We should definetely just keep one.

Snippets don't seem to have region tags, so they might not be used in docs. If we can confirm that I say we get rid of Snippets.

Reply:
I am porting the Samples over to be Snippets, then I will delete the Samples. I don't want to break things in the meantime, so doing it as multiple steps.

Reply:
Oh I see, makes sense. So maybe don't make any changes to Samples if they are staying there temporarily while you port them to Snippets.